source/_docs/automation/templating.markdown

```
@@ -54,26 +60,50 @@ Inherites template variables from [event](#event) or [state](#state) template ba
+| `trigger.platform` | Hardcoded: `event`
 | `trigger.event` | Event object that matched.
 | `trigger.event.event_type` | Event type.
 | `trigger.event.data` | Optional event data.
 
+### Geolocation
+
+These are the properties available for a [Geolocation trigger](/docs/automation/trigger/#geolocation-trigger). 
+
+| Template variable | Data |
+| ---- | ---- |
+| `trigger.platform` | Hardcoded: `geo_location`
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)